README.md

```diff
@@ -100,6 +110,13 @@ final class AppDelegate: ApplicationDelegate {
 	where `libpython3.10.so` is currently installed version and libpython3.8.so.1.0 is filename against which Swifts LLDB was built.
   </details>
 After the necessary dependencies were set up just add this package in your SwiftPM manifest file as a dependency and add **AppKid** product as a dependency to your target:
+eg:
+```bash
+ mkdir MyApp
+ cd MyApp
+ swift package init --type executable
+```
+Then replace the generated Package.swift in this folder with something like the sample below:
```

Comment:
This is a *very* obvious first step in making a swift project. Unfortunately this repo contains software in it's 0.0.*-alpha version which imposes requirements to some knowledge of how swift packages work

Reply:
I know, but its a couple of lines that would have helped *me* if they were there. [ as a new swift user, and a new AppKid wanta-be ] .  :-)

Reply:
I understand your struggle there, yet README provides a link to swift.org as part of the very first step of installation. Swift is the root tool for this repository, literally all actions are done via `swift ...` commands in one way or another. Even utility scripts are just calling swift.
The problem you described is really "how to bootstrap a new swift project" which is never intended to be solved by any code in this repo. For example, [Vapor](https://vapor.codes/), the server-side swift framework, actually ships a bootstrap tool to do that for new projects that want to use the framework. In my opinion this is conceptually wrong (tho I have a private tool that looks like an example from my [other project](https://github.com/smumriak/Cuisine)) and the only way to correctly fix this problem is for the person "on the other side" to go thru "Getting Started" section on https://www.swift.org/